### PR TITLE
Fusion bookmarks module

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Type of change

Remove options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read.
-   [ ] Documentation and comments is added where needed.
-   [ ] My code is covered by tests.
-   [ ] UX has reviewed relevant UI contributions.

### CODE REVIEW

File: packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/parseQueryParameters.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { bookmarksApi } from './bookmarksApi';
+import { validateBookmark } from './validateBookmark';
+
+export async function checkForBookmarkId() {
```
Comment (Noggling) [inline packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/parseQueryParameters.ts:4]: Code doc takk ;)

File: packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/getClientAsync.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { IHttpClient } from '@equinor/fusion-framework-module-http';
+//Re-declares window object
+import '@equinor/fusion-framework';
+
+export async function getFusionClient(): Promise<IHttpClient> {
```
Comment (Noggling) [inline packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/getClientAsync.ts:5]: code documentation

**Comment by Noggling** [comment]
Leg til dette Completes: AB#FILL_IN_YOUR_ISSUE_ID

**Comment by Noggling** [comment]
Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

**Comment by Noggling** [comment]
Dette ser ut til å trenge noen tester ;)

File: packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/handleBookmarksFromPortal.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	res: (value: unknown) => void,
+	rej: (reason?: string) => void
+): void => {
+	//TODO: Validate action type
+	/**Dont reject if not status because multiple events can be fired */
+	if (!isStatus(ev.data)) return;
+	if (ev.data.status === 'Success') {
+		res(ev.data);
+	} else {
+		rej('Failed');
```
Comment (Gustav-Eikaas) [inline packages/workspace-fusion-modules/src/lib/modules/fusionBookmarksModule/utils/handleBookmarksFromPortal.ts:88]: ev.data

**Pull request closed without merging**